app/views/access_requests/new.html.erb

```diff
@@ -19,6 +19,14 @@
       </div>
     </fieldset>
 
+    <div class="form-group">
+      <label class="col-lg-2 control-label">Projects</label>
+      <div class="col-lg-4">
+        <%= select_tag :project_ids, options_from_collection_for_select(Project.all.order(:name), 'id', 'name'),
```

Comment:
In general, it's an anti-pattern to make database calls in the view layer.  Better to load the data in the controller and set it to an instance variable.